hack/gen-content.sh

```diff
@@ -176,14 +178,16 @@ gen_link() {
 
   # If it was previously an "external" link but now local to the contributor site
   # update the link by trimming the url portion.
-  if echo "$generated_link" | grep -q -i -E "[http|https]://(git.k8s.io|(www\.)?github.com/kubernetes)"; then
+  # Detection is a  problem for non kubernetes repos.
+  # TODO: Support other external repos
+  if echo "$generated_link" | grep -q -i -E "https?:\/\/((sigs|git)\.k8s\.io|(www\.)?github\.com\/(kubernetes(-(client|csi|incubator|sigs))?|cncf))"; then
```

Comment:
@mrbobbytables Can we add a comment here regarding the regex as you mentioned in the issue ?